.github/workflows/tests.yml

```diff
@@ -2,9 +2,9 @@ name: tests
 
 on:
   push:
-    branches: ["master"]
+    branches: ["main"]
   pull_request:
-    branches: ["master"]
+    branches: ["main"]
```

Comment:
Seems like you must've changed the default branch name a while back, but never updated the GitHub action.